pkg/cmd/release/registry.go

```diff
@@ -209,4 +238,6 @@ func printTreeWorkflow(o *options, name string, w test, level uint) {
 	printTreeSteps(o, w.Test, level+1)
 	fmt.Println("post:")
 	printTreeSteps(o, w.Post, level+1)
+	fmt.Println("observers:")
+	printTreeObservers(w.Observers, level+1)
```

Comment:
```
$ go run ./cmd/release -C ~/src/release/ registry workflow --tree
workflow: 3scale-ipi-aws
…
observers:
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x19dddfc]
```

This cannot be done unconditionally.

Reply:
Also, tests and individual steps can declare observers (no workflow currently declares one in openshift/release@df4b5732d9d).  And I'd expect to get just the names normally and a full dump with `--resolve`, as we do for regular steps.

Reply:
If you'd like, remove this and we can merge just the listing part now, which looks good.  Then we can add this and the rest later.